mergify_cli/__init__.py

```diff
@@ -28,16 +29,14 @@
 import rich
 import rich.console
 
-from mergify_cli.commit_msg_hook import COMMIT_MSG_HOOK
-
 
 try:
     VERSION = importlib.metadata.version("mrgfy")
 except ImportError:
     # https://pyoxidizer.readthedocs.io/en/stable/oxidized_importer_behavior_and_compliance.html#importlib-metadata-compatibility
     VERSION = "0.1"
 
-CHANGEID_RE = re.compile(r"Change-Id: (I[0-9a-z]{40})")
+CHANGEID_RE = re.compile(r"Mergify-cli stack change-Id: (I[0-9a-z]{40})")
```

Comment:
this is quite ugly

Reply:
What do you propose?

Reply:
I don't even know the rational for changing it TBH

Reply:
To better identify that this id is related to Mergify CLI since we add it to the body. But maybe it's not useful and 'Change-Id' is enough. WDYT?